src/main/java/analyzer/exercises/secrets/ImplementOperator.java

```diff
@@ -0,0 +1,36 @@
+package analyzer.exercises.secrets;
+
+import analyzer.Comment;
+
+import java.util.Map;
+
+/**
+ * @see <a href="https://github.com/exercism/website-copy/blob/main/analyzer-comments/java/secrets/implement_operator.md">Markdown Template</a>
+ */
+class ImplementOperator extends Comment {
```

Comment:
Shall we name it `UseBitwiseOperator` instead?

1. Students don't implement the operators, they use them
2. I think the comment copy should mention the words "bitwise operator" so it makes sense to include this scope in the comment name.

Reply:
Sure, I agree! Done